docs/metadata.md

```diff
@@ -77,3 +77,24 @@ Cozy-Client will:
 
 ⚠️ These values will override any values that your document may already have.
 
+### Specific case of io.cozy.files
+
+`io.cozy.files` attribut cozyMetadata is protected by cozy-stack and can't be saved directly.
+
+This is the case for :
+* `cozyMetadata.sourceAccount`
+* `cozyMetadata.sourceAccountIdentifier`
+
+Since cozy-client 34.11.0, you can use a direct attribut in the document to save it.
```

Comment:
I'm not sure it's worth specifying the version for every change, the doc is supposed to be relevant only for the latest version.

Reply:
Reformulation suggestion:
To save those attributes, you can pass them directly to the `save` method, like this:

Reply:
Yeah, thank's, I was not sure too. Changed